cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/DocumentService.cs

```diff
@@ -100,5 +100,28 @@ public async override Task<GetDocumentsByTypeForUsersReply> GetDocumentsByTypeFo
 
             return result;
         }
+
+        public async override Task<GetDriverAndCaseDocumentsReply> GetDriverAndCaseDocuments(GetDriverAndCaseDocumentsRequest request, ServerCallContext context)
+        {
+            var result = new GetDriverAndCaseDocumentsReply();
+
+            try
+            {
+                // var loginIds = request.LoginIds.Select(Guid.Parse);
```

Comment:
you will be able to change this in a future PR now, since my loginIds claim is now merged